apps/web/pages/api/book/event.ts

```diff
@@ -669,6 +670,14 @@ async function handler(req: NextApiRequest) {
   let booking: Booking | null = null;
   try {
     booking = await createBooking();
+    // Sync Services
+    await syncServiceUpdateWebUser(
+      currentUser &&
+        (await prisma.user.findFirst({
+          where: { id: currentUser.id },
+          select: { id: true, email: true, name: true, plan: true, username: true },
+        }))
+    );
```

Comment:
Updating sync services when a new booking is created, in order to make it update last booking custom field.